docs/src/components/pricing/FAQ.tsx

```diff
@@ -63,6 +58,34 @@ const faqData = [
       </React.Fragment>
     ),
   },
+  {
+    summary: 'Why do we must license developers not using the software directly?',
+    detail: (
+      <React.Fragment>
+        Our pricing model requires all developers working on a project using MUI X Pro or Premium to
+        be licensed. This is designed to make it easier for your engineers to use the software
+        without having to constantly verify if they have the right number of seats. It aims to
+        replicate one of the great properties open-source licenses have: simplicity.
```

Comment:
```suggestion
```

Remove the gobbledegook

Reply:
Agree, I have tried to rework it to be simple and clear.